File: PHP/Exercicio08.php
<?php
echo "Digite a Distancia a ser Percorrida: ";
$distancia = fgets(STDIN); 
echo "Digite o Consumo Médio do Automóvel: ";
$consumo = fgets(STDIN); 

echo "Litros Necessarios: ".(floatval($distancia)/floatval($consumo))."L";

?>

File: Java/Exercicio08.java
package Java;

import java.util.Scanner;

public class Exercicio07 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite a Distancia a ser Percorrida: ");
        double distancia = scanner.nextDouble();
        System.out.print("Digite o Consumo Médio do Automóvel: ");
        double consumo = scanner.nextDouble();

        System.out.print("Litros Necessarios: "+distancia/consumo+"L");
        scanner.close();
    }
}
